src/test/java/org/ow2/proactive/connector/iaas/rest/InfrastructureRestTest.java

```diff
@@ -72,12 +72,18 @@ public void init() {
                                                                                        "openstack",
                                                                                        "endPoint",
                                                                                        "userName",
-                                                                                       "password");
+                                                                                       "password",
+                                                                                       null,
+                                                                                       null,
+                                                                                       null);
```

Comment:
can you possible overload this method, so we no need to pass 3 nulls in the end?

Reply:
Done, thanks